app/models/basket.rb

```diff
@@ -1,3 +1,7 @@
 class Basket < ApplicationRecord
   has_many :goods, dependent: :destroy
+
+  def handle_file(file)
```

Comment:
In my opinion, this method does not belong to the basket class, because all it does is it splits any file by line.

In this case you could extract a class: https://refactoring.guru/extract-class